scripts/environment/prepare.sh

```diff
@@ -8,7 +8,7 @@ rustup default "$(cat rust-toolchain)"
 rustup component add rustfmt
 rustup component add clippy
 rustup target add wasm32-wasi
-rustup run stable cargo install cargo-deb --version '^1.28.0'
+rustup run stable cargo install cargo-deb --git https://github.com/mmstick/cargo-deb.git
```

Comment:
Could we pin to a SHA here until the next release? I believe you can use `--rev`.